src/mutations/user/payTo.ts

```diff
@@ -220,6 +220,7 @@ const resolver: MutationToPayToResolver = async (
           currency: PAYMENT_CURRENCY.USDT,
           purpose: TRANSACTION_PURPOSE[purpose],
         })
+      payToByBlockchainQueue.payTo({ txId: transaction.id })
```

Comment:
Since this queue job needs to [wait](https://github.com/thematters/matters-server/pull/2871/files#diff-345232780656783e4918d3a2f53a196e3e2b2998a8cad6f0e152d5cf39bc11d8R218) for the transaction to be confirmed, it's better to [delay](https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#:~:text=delay%3A%20number%3B%20//%20An%20amount%20of%20milliseconds%20to%20wait%20until%20this%20job%20can%20be%20processed.%20Note%20that%20for%20accurate%20delays%2C%20both) (128 blocks / aprox. 5 mins) the job run when it's being added to queue, otherwise, there will be lots of jobs stuck in the queue.